README.md

```diff
@@ -63,19 +83,96 @@ To enable pgvectorscale:
     CREATE EXTENSION IF NOT EXISTS vectorscale CASCADE;
     ```
 
-   The `CASCADE` automatically installs the dependencies.
+   The `CASCADE` automatically installs `pgvector`.
+
+1. Create a table with an embedding column. For example:
+
+    ```postgresql
+    CREATE TABLE IF NOT EXISTS document_embedding  (
+        id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+        document_id BIGINT FOREIGN KEY(document.id)
```

Comment:
Do you think the FK constraint will trip people up since we don't include a definition of the document table?

Reply:
good catch